verilog/analysis/checkers/struct_union_name_style_rule.cc

```diff
@@ -81,13 +83,38 @@ void StructUnionNameStyleRule::HandleSymbol(const verible::Symbol& symbol,
     }
     const auto* identifier_leaf = GetIdentifierFromTypeDeclaration(symbol);
     const auto name = ABSL_DIE_IF_NULL(identifier_leaf)->get().text();
-    if (!verible::IsLowerSnakeCaseWithDigits(name) ||
+    // get rid of the exceptions specified by user
+    std::string name_filtered;
+    if (exceptions_.size()) {
```

Comment:
Just to figure out if there is something in there, we should use `empty()`. So `if (!exceptions_.empty())` in this case.